development/hooks-generator-php-8-5/Makefile

```
@@ -0,0 +1,46 @@
+# Default branch to checkout (can be overridden with make download BRANCH=your-branch)
+WORDPRESS_BRANCH ?= 7.0-branch
+BBPRESS_BRANCH ?= 2.6
+
+download_wordpress:
```

checkmake (0.3.2)

[warning] Target body for "download_wordpress" exceeds allowed length of 5 lines (7).

(maxbodylength)